lib/brine/step_definitions/request_construction.rb

```diff
@@ -12,6 +13,10 @@
   add_header(header, value)
 end
 
+When (/^the request credentials are set for basic auth user `([^`]*)` and password `([^`]*)`$/) do |user, password|
+  add_header('Authorization', 'Basic ' + Base64.strict_encode64(user + ':' + password))
```

Comment:
Prefer string interpolation over concatenation. It's likely worth splitting this into two lines where the encoded value is assigned to a temporary value.